src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -525,9 +606,9 @@ static void MX_GPIO_Init(void)
     HAL_GPIO_Init(USER_Btn_GPIO_Port, &GPIO_InitStruct);
 
     /*Configure GPIO pins : wheel_motor_back_right_esf_Pin
```

Comment:
Super Nit: Add a space before "Configure" to keep it consistent with the rest of the comments.

Reply:
this is autogenerated, unfortunately we can't change it